apps/api-extractor/src/extractor/Extractor.ts

```diff
@@ -439,7 +439,7 @@ export class Extractor {
       } else {
         this._monitoredLogger.logVerbose(`The "mainDtsRollupPath" is: ${mainDtsRollupPath}`);
 
-        if (!path.isAbsolute(mainDtsRollupPath)) {
+        if (path.isAbsolute(mainDtsRollupPath)) {
```

Comment:
>if (path.isAbsolute(mainDtsRollupPath)) { [](start = 8, length = 41)

Cool, thanks for fixing this!

BTW I recall there were some design issues with the way these paths are configured. When SPFx enabled .d.ts rollups they solved it using some temporary workarounds, and we were planning to come back and improve it later.  So if you encounter any other issues/questions please let me know.